packages/app/src/DOMPlayer.tsx

```diff
@@ -513,17 +513,18 @@ export default function DOMPlayer({
             No replay available for this session, most likely due to this
             session starting and ending in a background tab.
           </div>
-        ) : null}
-        <div
-          ref={wrapper}
-          className={cx(styles.domPlayerWrapper, 'overflow-hidden', {
-            'd-none': isLoading || isBuffering,
-            started: (replayer.current?.getCurrentTime() ?? 0) > 0,
-            [styles.domPlayerWrapperPaused]: playerState === 'paused',
-          })}
-        >
-          <div className="player rr-block" ref={playerContainer} />
-        </div>
+        ) : (
```

Comment:
@ernestii what's the reason of this change? This seems to break the replayer at times since `playerContainer` ref will be null in the `useEffect` above, seems to introduce a race condition where session replays will incorrectly show up as "no replay available"